### PR TITLE
Add new plugin to Starlight (`resources/plugin.mdx`)

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description

- add new plugin called `starlight-sidebar-plugin-dropdown`
- Starlight plugin to split your docs page into multiple subpages and switch between them with a dropdown menu in the sidebar.

**1000 thanks to @HiDeoo the actual idea for such a plugin was their idea! (I just changed the visuals a bit)**

Try out the plugin [here](https://starlight-sidebar-topics-dropdown.trueberryless.org/).

### CODE REVIEW

**Review by apatel369 — APPROVED** [review]
LGTM

File: docs/src/content/docs/resources/plugins.mdx [hunk 1]
```diff
@@ -93,6 +93,11 @@ Extend your site with official plugins supported by the Starlight team and commu
 		title="starlight-site-graph"
 		description="Add an interactive site graph inside your page's sidebar."
 	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-sidebar-topics-dropdown"
+		title="starlight-sidebar-topics-dropdown"
+		description="Split your docs page into multiple subpages and switch between them with a dropdown menu in the sidebar."
+	/>
```
Comment (delucis) [inline docs/src/content/docs/resources/plugins.mdx:100]: What do you think about also adding the original plugin as part of this PR? Just realised it hasn’t been added here yet.

```suggestion
	<LinkCard
		href="https://github.com/HiDeoo/starlight-sidebar-topics"
		title="starlight-sidebar-topics"
		description="Starlight plugin to split your documentation into different sections, each with its own sidebar."
	/>
	<LinkCard
		href="https://github.com/trueberryless-org/starlight-sidebar-topics-dropdown"
		title="starlight-sidebar-topics-dropdown"
		description="Split your docs page into multiple subpages and switch between them with a dropdown menu in the sidebar."
	/>
```

**Review by delucis — APPROVED** [review]
Thanks @trueberryless! Just left one suggestion 🙌

File: docs/src/content/docs/resources/plugins.mdx (hunk 1 above)
Comment (trueberryless) [reply docs/src/content/docs/resources/plugins.mdx:100]: I love this suggestion! I wanted to add it originally, but didn't want to just take the credit and let HiDeoo do it. Actually I contacted him privately and asked him about it. I'll give him the option that we can include it here right away, let's wait for his answer though...

Comment (HiDeoo) [reply docs/src/content/docs/resources/plugins.mdx:105]: Amazing idea, I totally forgot to do that, thanks you both 🙌

**Comment by trueberryless** [comment]
I updated the description so now it fits the new added commits.

Good job to everyone and a huge thanks again @HiDeoo for creating such a beautiful plugin for Starlight!

**Pull request merged**